AzureAutomationAuthoringToolkit/AzureAutomationAuthoringToolkitInner.psm1

```diff
@@ -170,22 +177,10 @@ function Install-AzureAutomationIseAddOn {
     else {
          Write-Verbose "AzureAutomationAuthoringToolkit: Content already present. Not adding any content."
     }
-
```

Comment:
Oh, I see. Because we are doing the copy on module import already, it will be handled at that time. OK. should still rename the cmdlet though.